CHANGELOG.md

```diff
@@ -1,6 +1,64 @@
+# 1.7.0-rc01 (September 2024)
+_Changes since 1.7.0-beta02_
+
+## Highlights
+### Web
+- [`skiko.js` is redundant in case of K/Wasm Compose for Web applications and it can be removed from index.html files to not load redundant files](https://github.com/JetBrains/compose-multiplatform/pull/5134). `skiko.js` will be removed from the k/wasm distribution in the future releases. `skiko.js` is still needed in case of K/JS Compose for Web apps
+
+## Features
+### Multiplatform
+- [Support Kotlin 1.9.25](https://github.com/JetBrains/compose-multiplatform/pull/5141)
+
+### Desktop
+- _(prerelease fix)_ [The `decoration` parameter added to `Window` and `DialogWindow`, and the APIs related to, it are now marked as experimental](https://github.com/JetBrains/compose-multiplatform-core/pull/1561)
+
+### Gradle Plugin
+- [New `compose.material3AdaptiveNavigationSuite` shortcut in the gradle plugin](https://github.com/JetBrains/compose-multiplatform/pull/5133)
+
+## Fixes
+### Android
+- _(prerelease fix)_ [Fix "Compose UI test error on android: No static method forceEnableAppTracing"](https://github.com/JetBrains/compose-multiplatform-core/pull/1564)
+- _(prerelease fix)_ [Fix "Android target depends on prerelease versions"](https://github.com/JetBrains/compose-multiplatform-core/pull/1564)
+
+### Multiple Platforms
+- _(prerelease fix)_ [Fix possible infinity invalidation loop triggered by `GraphicsLayer.record`](https://github.com/JetBrains/compose-multiplatform-core/pull/1555)
+
+### iOS
+- _(prerelease fix)_ [Fix "ListDetailPaneScaffold from material3-adaptive throws ArrayIndexOutOfBoundsException"](https://github.com/JetBrains/compose-multiplatform-core/pull/1548)
+- _(prerelease fix)_ [Fix compose scene scale to match iOS natural screen scale](https://github.com/JetBrains/compose-multiplatform-core/pull/1547)
```

Comment:
compose scene is internal primitive, it says nothing to customers

Reply:
Done